zebra-consensus/src/script.rs

```diff
@@ -50,9 +50,13 @@ impl<ZS> Verifier<ZS> {
 /// A script verification request.
 #[derive(Debug)]
 pub struct Request {
-    /// A cached transaction, in the format required by the script verifier FFI interface.
+    /// Ideally, this would supply only an `Outpoint` and the unlock script,
+    /// rather than the entire `Transaction`, but we call a C++
+    /// implementation, and its FFI requires the entire transaction.
+    ///
+    /// This causes quadratic script verification behavior, so
+    /// at some future point, we need to reform this data.
```

Comment:
This comment reverts #1680, but #1680 is correct

Reply:
oh yea, unintentional. I "squashed" my commits by doing `git reset --soft main` after merging main in and it looks like #1680 merged after so im guessing it just tripped up the merge resolution because of the times? Not really sure how it broke but yea, fixing